### PR TITLE
Add nsis-reference extension

### PR DESCRIPTION
**Pull request by idleberg**
## Description

Adds actions to look up the NSIS documentation

## Type of change

- New extension

## Screencast

https://user-images.githubusercontent.com/1504938/151202229-d65dd2d7-730f-4ec5-8f9a-71d424b95d56.mp4

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)

### CODE REVIEW

File: extensions/nsis-reference/package.json [hunk 1]
```diff
@@ -0,0 +1,37 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "nsis-reference",
+  "title": "NSIS Reference",
+  "description": "",
```
Comment (pernielsentikaer) [inline extensions/nsis-reference/package.json:5]:
```suggestion
  "description": "Search NSIS references",
```

File: extensions/nsis-reference/package.json [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+  "icon": "command-icon.png",
+  "author": "idleberg",
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "index",
+      "title": "NSIS Reference",
```
Comment (pernielsentikaer) [inline extensions/nsis-reference/package.json:12]:
```suggestion
      "title": "Search References",
      "subtitle": "NSIS",
```

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
Hi @idleberg 👋 

Looks like an awesome extension you're building 💪 I left some feedback for you to work with, also, it looks like you haven't formatted it with prettier, could you also look into this?

Request a new review when you are ready, feel free to contact me here or at [Slack](https://raycastcommunity.slack.com/) if you have any questions.

**Comment by idleberg** [comment]
Thanks @pernielsentikaer. I made the requested changes.

**Review by pernielsentikaer — APPROVED** [review]
Hi @idleberg 👋 

Thanks for your contribution, much appreciated 💪 I'm going to accept it and then my colleague will do a final review before it's finally merged.

**Comment by sxn** [comment]
🎉

**Pull request merged**